test/test-checks.el

```diff
@@ -24,7 +24,7 @@
       (checkdoc-file file))
     (when (get-buffer "*Warnings*")
       (message "Failing due to checkdoc warnings...")
-      (kill-emacs 1)))
+      (kill-emacs 1))))
 
-  (when (apply #'check-declare-files files)
-    (kill-emacs 1)))
+  ;; (when (apply #'check-declare-files files)
```

Comment:
Why did you comment this out?

Reply:
hahha @bbatsov this is the fastest anyone has commented on one of my pull requests. Didn't submit my comments in time :)

Reply:
🤣

Reply:
See https://github.com/clojure-emacs/clojure-mode/pull/532#discussion_r301415357